pkgs/development/python-modules/scikit-misc/default.nix

```diff
@@ -54,9 +48,32 @@ buildPythonPackage rec {
     wheel
   ];
 
-  buildInputs = [
+  propagatedBuildInputs = [
     numpy
-    openblas
+  ];
+
+  buildInputs = [
+    blas
+    lapack
+  ];
+
+  mesonFlags = [
+    "-Dblas=${blas.pname}"
+    "-Dlapack=${lapack.pname}"
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  # have to move things around to avoid import errors when loading the test
+  preCheck = ''
+    mv skmisc/loess/tests .
+    mv skmisc skm
+  '';
```

Comment:
Does this work?
```suggestion
  pytestFlagsArray = [ "--import-mode=importlib" ];
```

Reply:
this still results in the error, unfortunately.

```
==================================== ERRORS ====================================
_____________________ ERROR collecting tests/test_loess.py _____________________
skmisc/__init__.py:15: in <module>
    from skmisc.__config__ import show as show_config  # noqa: F401
E   ModuleNotFoundError: No module named 'skmisc.__config__'

During handling of the above exception, another exception occurred:
/nix/store/ghh2dgix6cfhwglxab5aar02y4qa73xb-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/runner.py:341: in from_call
    result: Optional[TResult] = func()
...
tests/test_loess.py:9: in <module>
    from skmisc.loess import loess, loess_anova
skmisc/__init__.py:20: in <module>
    raise ImportError('\n\n'.join([err.message, msg]))  # type: ignore
E   AttributeError: 'ModuleNotFoundError' object has no attribute 'message'
```

Reply:
could do something like this.
```
  preCheck = ''
    export HOME="$(mktemp -d)"
    cp -a skmisc/loess/tests "$HOME"
    cd "$HOME"
  '';
```
it is different but not clear it is any better. setuptools has a flag `--inplace` that solved similar problems but that does not seem to work when added to pypaBuildFlags.

Reply:
this works tho.
```nix
  # can not run tests from source directory
  preCheck = ''
    export HOME="$(mktemp -d)"
    cd
  '';

  pytestFlagsArray = [
    "--pyargs skmisc"
  ];
```